src/components/App.js

```diff
@@ -4,16 +4,26 @@ import NewMember from './newMember';
 import '../App.css';
 import { members } from '../data';
 import MemberList from './memberList';
+import { useState } from 'react';
 
 export default function App() {
+  const [listOfMembers, setListOfMembers] = useState(members);
+
+  function handleAddNewMember(memberName) {
+    setListOfMembers([...listOfMembers, {
```

Comment:
This should probably have an updater function so you don't end up with a stale state.
```js
setListOfMembers(members => [...members, {
```